package.json

```diff
@@ -1,8 +1,12 @@
 {
   "name": "root",
   "private": true,
+  "workspaces": [
+    "packages/*"
+  ],
   "scripts": {
-    "dev": "lerna run dev",
+    "dev": "lerna run --parallel dev",
```

Comment:
This dev script will run the "dev" scripts for each package (so both Next apps, as well as the new `rollup --watch` mode for the component library.)

There is also a script to run without the rollup watch since it does require a fair bit of memory and increase hot reload times.  using storybook to work on the components is a preferable option